common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -41,6 +41,25 @@
   @JsonProperty("password")
   private PasswordProvider passwordProvider;
 
+  public MetadataStorageConnectorConfig(){
+    // NOOP
```

Comment:
can we just call this(null, null, null, ...) to make it clear the new constructor is the main one?